src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.tsx

```diff
@@ -0,0 +1,105 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { Avatar, DataList, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../../core';
+
+export interface IAssetDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The source of the logo for the token.
+     */
+    logoSrc?: string;
+    /**
+     * The name of the Token.
+     */
+    tokenName?: string;
+    /**
+     * The symbol of the Token.
+     */
+    symbol?: string;
+    /**
+     * The amount of the Token.
+     */
+    amount?: number | string;
+    /**
+     * The price of the Token.
+     */
+    USDAmount?: number | string;
```

Comment:
The comment & property name are confusing, is this the USD amount based on the amount or is this the token price? I would only add a property like `usdPrice` that refers to the price of the token and eventually calculate the amount * usdPrice on the component